tests/specs/scss/functions.scss

```diff
@@ -30,6 +30,34 @@
     @include assert-equal($config, $expect, "Check if map is correct.");
   }
 
+  // Testing set functions
+  @include test("config-set [function] Set a map as vlaue.") {
+    @include config-set("map", ("bar": "foo", "foo": "bar"));
+
+    $expect: ("bar": "foo", "foo": "bar");
+    $config: config-get("map");
+
+    @include assert-equal($config, $expect, "Check if map is correct.");
+  }
+
+  // Testing set functions
+  @include test("config-set [function] Set a map variable as vlaue.") {
+    $color-black: #000 !default;
+    $color-white-bg: #fff !default;
+
+    $color: (
+        fg: $color-black,
+        bg: $color-white-bg
+    ) !default;
+
+    @include config-set("map-variable", $color);
+
+    $expect: (fg: #000, bg: #fff);
```

Comment:
Color literals like `#000` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.